packages-common/pluggable-widgets-generator/generators/app/templates/pluggable/web/fullTemplateTs/src/components/Alert.tsx.ejs

```diff
@@ -0,0 +1,15 @@
+import { FunctionComponent, createElement } from "react";
+import classNames from "classnames";
+
+export interface AlertProps {
+    message?: string;
+    className?: string;
+    bootstrapStyle: "default" | "primary" | "success" | "info" | "inverse" | "warning" | "danger";
+}
+
+export const Alert: FunctionComponent<AlertProps> = (props) =>
```

Comment:
Alert is not used in the example code, so imho can be removed from scaffolding. It seems to be frequently used by widgets, then we can decide to include it as well in `pluggable-widgets-tools`. What do you think @diego-antonelli ?

Reply:
If we will start to deliver components & helper classes then YES

Reply:
Feedback reference included in a new story, so will be picked up in a follow up PR.